glom/core.py

```diff
@@ -911,6 +904,50 @@ def _t_eval(target, _t, scope):
 _T_PATHS[T] = (T,)
 _T_PATHS[S] = (S,)
 UP = make_sentinel('UP')
+ROOT = make_sentinel('ROOT')
+
+
+class Let(object):
+    """
+    Let("var").over(subspec) -- assigns var to
+    scope that subspec executes under
+
+    other form is Let(foo=spec1, bar=spec2).over(subspec)
+    """
+    def __init__(self, *a, **kw):
+        if a:
```

Comment:
i don't think we want positional arguments, Let(var=T) and Let("var") are the same number of chars so I'm going to defer it for now.

Reply:
fixed